### PR TITLE
Fixes #2106: Renames import statements

### LINKED ISSUE #2106: Import statements not changed
**I'm submitting a ...**
- [x] bug report
- [ ] feature request

**Description :**

@abishekvashok #2074 api imports should be changed to backend. 

**Screenshots**

Screenshot from 2019-08-18 23-46-37

### PR DESCRIPTION
**Pull request by abishekvashok**
Fixes #2106

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] I have added necessary documentation (if appropriate)

#### Changes proposed in this pull request:

Renames all import statements to make backend working again

### CODE REVIEW

File: backend/static/badges/4x3onA4.svg [hunk 1]
```diff
@@ -20,61 +20,61 @@
...
...-ligatures:normal;font-variant-caps:normal;font-variant-numeric:normal;font-feature-settings:normal;text-align:start;letter-spacing:0px;word-spacing:0px;writing-mode:lr-tb;text-anchor:start;fill:#ffffff;fill-opacity:1;stroke:none;stroke-width:0.26458332" font-family="font-style:normal;font-variant:normal;font-weight:normal;font-stretch:normal;font-size:5.64444447px;line-height:1.25;font-family:Helvetica;-inkscape-font-specification:'sans-serif, Normal';font-variant-ligatures:normal;font-variant-caps:normal;font-variant-numeric:normal;font-feature-settings:normal;text-align:start;letter-spacing:0px;word-spacing:0px;writing-mode:lr-tb;text-anchor:start;fill:#ffffff;fill-opacity:1;stroke:none;stroke-width:0.26458332"><tspan style="font-style:normal;font-variant:normal;font-weight:normal;font-stretch:normal;font-size:5.64444447px;font-family:sans-serif;-inkscape-font-specification:'sans-serif, Normal';font-variant-ligatures:normal;font-variant-caps:normal;font-variant-numeric:normal;font-feature-settings:normal;text-align:start;writing-mode:lr-tb;text-anchor:start;fill:#ffffff;font-family:Helvetica" y="273.30194" x="6.022738" id="tspan105" sodipodi:role="line">Person_1_5</tspan></text>
```
Comment (kushthedude) [inline backend/static/badges/4x3onA4.svg]: @abishekvashok What change you made in here

Comment (abishekvashok) [reply backend/static/badges/4x3onA4.svg]: Don't need those changes, it occured while testing. Let me reset them

**Comment by abishekvashok** [comment]
@kushthedude done

**Comment by kushthedude** [comment]
@abishekvashok Do you think it was a good option to rename api to backend ? Import statements make much more sense when they were making with api. Still I am asking for your personal choice on it.  What you say?

**Comment by abishekvashok** [comment]
@kushthedude yes I think it is logical as for the personal choice.

As a collective choice please see the simplify documentation issue.

**Comment by abishekvashok** [comment]
Self mergin as it is very important

**Pull request merged**